### ISSUE TITLE
Dear engineer,Can silO and silO grain communicate with each other?

### ISSUE THREAD

**Issue by wz172** [dotnet/orleans#7828]
First, Thank you very much for answering my questions. 
        I've been watching silo for a while, and I know that the same silo can get Grain in silo via GrainFactory. Grain can be obtained from the Client for clusters with different silOS.Grain can be obtained from the Client after the end is connected to the cluster. But how to directly access Grain in one cluster and Grain in another silo. Can you give me an example? Thank you very much

**Comment by benjaminpetit** [comment]
Hi,

Let me rephrase to be sure I understand what you want:

A cluster is a set of silos that have the same configured `ClusterId`.
You have two clusters of silos, A and B. You want one client to be connected to both clusters, and this client has to be able to know to which cluster it is talking to.

Correct?

**Comment by wz172** [comment]
Yes.

The purpose is that the grain of two different Silos in the cluster can call each other.But instead of accessing grain from an out-of-code client, it is accessed from within the cluster.

I used the Redis component to form a cluster of heterogeneous SilOS. Is it possible to get the Grain interface objects in all clusters in one Grain, via silo's built-in hosted Iclusterclient object or IGrainFactory? Of course, I need to add the IGrain interface from the Silo of this Grain.

Thank you again for your answer, Your mind is sharp and clear. you`re very professional.
